pkg/reader/ccip.go

```diff
@@ -625,6 +625,17 @@ func (r *ccipChainReader) GetRMNRemoteConfig(
 	}, nil
 }
 
+func (r *ccipChainReader) isValidAddress(address []byte, contractName string, chain cciptypes.ChainSelector) bool {
```

Comment:
nit: if you return rather than logging this might be helpful for other functions as well.
```suggestion
func isValidAddress(address []byte, contractName string, chain cciptypes.ChainSelector) (bool, error) {
```

Reply:
I removed it completely for now.